server/src/test/java/io/crate/integrationtests/FulltextIntegrationTest.java

```diff
@@ -89,6 +94,17 @@ public void testSelectOrderByScore() throws Exception {
             "Time is an illusion. Lunchtime doubly so",
             "Would it save you a lot of time if I just gave up and went mad now?"
         );
+
+        // Match added column
+        execute("select added_indexed_col from quotes where match(added_indexed_col, ?) " +
+                "order by \"_score\" desc",
+            new Object[]{"time"}
+        );
+        assertThat(response).hasRows(
+            "Time is an illusion. Lunchtime doubly so",
+            "Would it save you a lot of time if I just gave up and went mad now?"
+        );
```

Comment:
Initially I thought test case in AddColumnTaskTest is enough since it fails on master and passes on this PR.

However, this test still fails, so maybe makes sense to keep it even after I make it pass.